packages/common/date/src/date-suggestion/domain/types/suggestion-strategy-map.ts

```diff
@@ -0,0 +1,5 @@
+import { SuggestionStrategyDescriptor } from '../interfaces/suggestion-strategy-descriptor';
+
+export type SuggestionStrategyMap<Keys extends string | symbol | number = string | symbol | number> = {
```

Comment:
`Keys extends string | symbol | number = string | symbol | number` reuse StrategyKey

Reply:
use it here `StrategyKey`